ovos_utils/sound/__init__.py

```diff
@@ -32,110 +33,86 @@ def _get_pulse_environment(config):
     else:
         return os.environ
 
-
-def play_acknowledge_sound():
-    """Acknowledge a successful request.
-
-    This method plays a sound to acknowledge a request that does not
-    require a verbal response. This is intended to provide simple feedback
-    to the user that their request was handled successfully.
-    """
+def _play_specific_sound(sound_name, bus=None):
     audio_file = resolve_resource_file(
```

Comment:
operates on paths, next thing on URIs

Reply:
This helper routine just factored out the 4*repeated code which takes a "partial path" which was hardcoded into the code. (see the single commit for more clarity - https://github.com/OpenVoiceOS/ovos-utils/pull/185/commits/5778b55e0e875f1646bc44970f0446c474059725)

Reply:
Right, we'll spin this off into its own ticket then. But does it actually work as-is? I'm in a remote, unable to spin up a bus to test this, and I can't remember if the URI-expecting functions will Just Work on a path

Reply:
let's keep this all to a follow up PR, @goldyfruit just noted some pulseaudio related issues aswel so this will need more work and this PR is already ballooning out of scope

this is the logic that was there before so any bug is not introduced in this PR

Reply:
https://github.com/OpenVoiceOS/ovos-utils/pull/187